### PR TITLE
Add verification step to AAS tutorial

### PR DESCRIPTION
**Pull request by JAB1305**

### CODE REVIEW

File: sdk/basyx/tutorial/tutorial_create_simple_aas.py [hunk 1]
```diff
@@ -132,3 +133,13 @@
...
     ]
 )
+
+##########################################################################
+# Step 4: Verify the Asset Administration Shell (AAS) and its components #
+##########################################################################
+# This step ensures that the AAS conforms to the rules and constraints defined by the AAS metamodel. The fields
+# themselves do not underlie any restriction.
+
+# We can use aas_core3.verification.verify(). This method returns an Iterator that we can collect into a list and
```
Comment (s-heppner) [inline sdk/basyx/tutorial/tutorial_create_simple_aas.py:143]: Can you expand here on what's inside the Iterator? Probably a list of problems that would occur?

**Review by s-heppner — CHANGES_REQUESTED** [review]

File: sdk/basyx/tutorial/tutorial_create_simple_aas.py (hunk 1 above)
Comment (JAB1305) [reply sdk/basyx/tutorial/tutorial_create_simple_aas.py:143]: Yes, exactly that. Maybe update the desription to something like this?

```
# We can use aas_core3.verification.verify(). This method returns an Iterator that we can collect into a list and
# for demonstration reasons assert it to be empty. Each item in this iterator represents a violation of the AAS
# meta-model rules or constraints (e.g. Missing required fields, Invalid types of fields, ...).
```

Comment (JAB1305) [reply sdk/basyx/tutorial/tutorial_create_simple_aas.py:143]: And do we maybe want to add this link to the official documentation?
https://aas-core30-python.readthedocs.io/en/latest/getting_started/verify.html

Comment (s-heppner) [reply sdk/basyx/tutorial/tutorial_create_simple_aas.py:143]: Yes, I like your text, and you can add: 

`For more information refer to the official aas-core documentation`. 

I don't think we should add a link to aas-core v3.0, considering the tutorial should stay valid even with newer versions of aas-core and then we will most definitely forget to update this link.

**Review by s-heppner — APPROVED** [review]

**Pull request merged**